packages/mobile/src/utils/zIndex.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/no-duplicate-enum-values */
 export enum zIndex {
   CHAT_TAIL = -10,
-  CHAT_BLAST_SUBHEADER = 1,
+  ONE = 1,
```

Comment:
why make this generic? the whole point was to have names here

Reply:
yeah ok fair